docusaurus.config.js

```diff
@@ -118,26 +118,45 @@ const config = {
                 },
               },
             footer: {
-                style: 'dark',
+                logo: {
+                    alt: 'WasmEdge logo',
+                    src: '/img/wasmedge_logo.svg',
+                    href: 'https://wasmedge.org/',
+                    height: 105,
+                    width: 205
+                  },
+                style: 'light',
                 links: [
                     {
-                        title: 'Docs',
+                        title: 'Resources',
                         items: [
                             {
-                                label: 'Develop',
-                                to: '/develop/overview',
+                                label: 'Github',
+                                to: 'https://github.com/WasmEdge/WasmEdge',
                             }, {
-                                label: 'Embed',
```

Comment:
Was there any reason for removing guides links?

Reply:
Yes, this is a mistake.